src/components/WalletModal/index.tsx

```diff
@@ -361,32 +311,30 @@ export default function WalletModal({
         <CloseIcon onClick={toggleWalletModal}>
           <CloseColor />
         </CloseIcon>
-        {walletView !== WALLET_VIEWS.ACCOUNT ? (
+        {connector === network ? (
```

Comment:
This should still be `walletView !== WALLET_VIEWS.ACCOUNT`, or else the "Connect a wallet" header will not be shown, and a left-arrow will be shown on the options menu that does nothing.